package.json

```diff
@@ -69,6 +69,7 @@
     "react-art": "^16.8.6",
     "react-native-actionsheet": "^2.4.2",
     "react-native-image-zoom-viewer": "^2.2.25",
+    "react-native-cached-image": "link:../react-native-cached-image",
```

Comment:
Need to publish forked package of this repo. Original repo is not React-native 0.60 compatible